PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -311,11 +339,11 @@
         }
 
         private func hasText(_ text: String?) -> Bool {
-            if let text, !text.isEmpty {
-                true
+            if let text = text, !text.isEmpty {
```

Comment:
`make format` does that, i think we are using different lint and format versions @ioannisj ?